resources/views/tournaments/view.blade.php

```diff
@@ -256,6 +256,57 @@
                     </div>
                 </div>
             @endif
+            {{--Videos--}}
+            <div class="bracket">
+            @if ($tournament->concluded)
+                <h5>
+                    <i class="fa fa-video-camera" aria-hidden="true"></i>
+                    Videos
+                </h5>
+                {{--Add video--}}
+                <button class="btn btn-primary btn-xs" id="button-add-videos"
+                        onclick="toggleVideoAdd(true)">
+                    <i class="fa fa-video-camera" aria-hidden="true"></i> Add videos
+                </button>
+                <button class="btn btn-primary btn-xs hidden-xs-up" id="button-done-videos"
+                        onclick="toggleVideoAdd(false)">
+                    <i class="fa fa-check" aria-hidden="true"></i> Done
+                </button>
+                <div id="section-add-videos" class="hidden-xs-up small-text">
+                    <hr/>
+                    <div class="p-b-1">
+                        Add a Youtube video
```

Comment:
Let's have this with normal styling and use the "small-text" class only for the form labels. I think it looks better that way.